src/test/java/org/killbill/billing/plugin/adyen/recurring/AdyenRecurringClient.java

```diff
@@ -0,0 +1,46 @@
+package org.killbill.billing.plugin.adyen.recurring;
```

Comment:
> Copyright Header for Test Classes?

Yup! We're using the [RAT plugin](http://creadur.apache.org/rat/apache-rat-plugin/) in the Travis profile, which would fail the build if they are missing.